controllers/chat/moveToSign.js

```diff
@@ -5,6 +5,7 @@ const {responseSuccess} = require('../../utils/Responses');
 const {CHANNEL_TYPE} = require('../../helpers/constants');
 const UsersFunction = require('../../databases/functions/users');
 const Getstream = require('../../vendor/getstream');
+const BetterSocialCore = require('../../services/bettersocial');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `BetterSocialCore` is new in this hunk. Ensure that all methods used from this module are correctly implemented and tested.
```diff
+ const BetterSocialCore = require('../../services/bettersocial');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->